tests/annotation_tests.ts

```diff
@@ -195,11 +201,11 @@ function harmonic(options: TestOptions, contextBuilder: ContextBuilder): void {
 function picking(options: TestOptions, contextBuilder: ContextBuilder): void {
   const ctx = contextBuilder(options.elementId, 500, 240);
 
-  ctx.setFont(Font.SANS_SERIF, FONT_SIZE);
-  const stave = new TabStave(10, 10, 450).addTabGlyph().setContext(ctx).draw();
+  ctx.setFont(Tables.lookupMetric('fontFamily'), FONT_SIZE);
+  const stave = new TabStave(10, 10, 450).addClef('tab').setContext(ctx).draw();
 
   const annotation = (text: string) =>
-    new Annotation(text).setFont(Font.SERIF, FONT_SIZE, FontWeight.NORMAL, FontStyle.ITALIC);
+    new Annotation(text).setFont(Tables.lookupMetric('Annotation.fontFamily'), FONT_SIZE, 'normal', 'italic');
```

Comment:
Same concern here. I think a developer using the Annotation class shouldn't have to call `Tables.lookupMetric('Annotation.fontFamily')`.

Shouldn't they just call `new Annotation(text)`? Should the font default to the one they chose for the text font (presumably earlier)?

Basically when a developer adds a new VexFlow score on a webpage, I imagine they should do two things:

1) set the music engraving font, and load the font file
2) set the text font, and load the font file if needed
3) ... everything else.

I've been looking at the SVG elements output by VexFlow 5 and I find it weird that the music symbol elements have fontFamily: "Bravura, Roboto Slab". I probably will file an issue for this... so we can investigate. 

I think long term, elements that are only musical glyphs should only have a music font stack. Elements that only have text glyphs should only have a text font stack (e.g., "Academico, Times New Roman, serif").

Elements that are for some reason mixed (both music and text symbols) are allowed to have a mixed font stack (e.g, "Bravura, Academico").

Reply:
same us above

Reply:
OK, we will revisit this font stack question once more parts are migrated.